#!/usr/bin/env python
PACKAGE = "robot_driver"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("linear_x", double_t, 0, "Linear velocity in x direction", 0.0, -5.0, 5.0)
gen.add("linear_y", double_t, 0, "Linear velocity in y direction", 0.0, -5.0, 5.0)
gen.add("linear_z", double_t, 0, "Linear velocity in z direction", 0.0, -5.0, 5.0)
gen.add("angular_x", double_t, 0, "Angular velocity in x direction", 0.0, -5.0, 5.0)
gen.add("angular_y", double_t, 0, "Angular velocity in y direction", 0.0, -5.0, 5.0)
gen.add("angular_z", double_t, 0, "Angular velocity in z direction", 0.0, -5.0, 5.0)
gen.add("transmission_frequency", double_t, 0, "Transmission frequency in Hz", 10.0, 1.0, 50.0)

exit(gen.generate(PACKAGE, "CmdVelAdjuster", "CmdVelAdjuster"))
